README.md

```diff
@@ -487,7 +487,7 @@ List of all available attributes can be found at [otel-webserver-module](https:/
 
 #### Using Nginx autoinstrumentation
 
-For `Nginx` autoinstrumentation, Nginx versions 1.22.0, 1.23.0, and 1.23.1 are supported at this time. The Nginx configuration file is expected to be `/etc/nginx/nginx.conf` by default, if it's different, see following example on how to change it. Instrumentation at this time also expects, that `conf.d` directory is present in the directory, where configuration file resides and that there is a `include <config-file-dir-path>/conf.d/*.conf;` directive in the `http { ... }` section of Nginx configuration file (like it is in the default configuration file of Nginx). You can also adjust OpenTelemetry SDK attributes. Example:
+For `Nginx` autoinstrumentation, Nginx versions 1.24.0 and 1.25.3 are supported at this time. The Nginx configuration file is expected to be `/etc/nginx/nginx.conf` by default, if it's different, see following example on how to change it. Instrumentation at this time also expects, that `conf.d` directory is present in the directory, where configuration file resides and that there is a `include <config-file-dir-path>/conf.d/*.conf;` directive in the `http { ... }` section of Nginx configuration file (like it is in the default configuration file of Nginx). You can also adjust OpenTelemetry SDK attributes. Example:
```

Comment:
Yes, I agree! How should that be reflected? Should I bump the major?

Only problem is that it can be confusing that the semantic version deviates from the upstream.

Reply:
You need to add a changelog entry with type `breaking_change`. Run `make chlog-new` and have a look at the added changelog record.

I'm wondering if we should actually have the operator pick the right version based on the nginx image version. WDYT @jaronoff97 @pavolloffay ?

Reply:
with the bump to 1.0.4 is all support for those versions dropped? We definitely should add a changelog calling out the dropping of support. I'm wondering if we should also add a compat matrix for the versions / images to let people know.